docs/guides/real-time/getting-started/websockets.md

```diff
@@ -57,7 +57,7 @@ connection.addEventListener('open', function () {
 });
 
 connection.addEventListener('message', function (message) {
-	const { data } = JSON.parse(message);
+	const { data } = message;
```

Comment:
The original example was indeed wrong, however `message.data` is still a string containing the JSON response.

The correct solution here is to parse `message.data`:
```suggestion
	const data = JSON.stringify(message.data);
```
https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/message_event#event_properties

Reply:
Did you mean? 😄 
```suggestion
	const data = JSON.parse(message.data);
```

Reply:
Lol, guess i didnt have enough coffee yet 😂

Reply:
😀
![monday](https://github.com/directus/directus/assets/5363448/03fc35ee-2601-456c-bfd5-8f2bc4ed63f2)